packages/lesswrong/components/questions/NewAnswerCommentQuestionForm.jsx

```diff
@@ -93,6 +93,8 @@ class NewAnswerCommentQuestionForm extends PureComponent {
           post={post}
           type="comment"
         />
+      case "question":
```

Comment:
Should we take the opportunity to also improve our answer/comment ontology? 

We had noticed at least some annoyance at people posting answers that should have been comments, so I think we should at least reverse the defaults there. 

We'd talked about "contribution" and "synthesis." Is that concept within spitting distance of done, enough that we can change the names here?